Earthfile

```diff
@@ -135,6 +135,9 @@ spaceros-artifacts:
   COPY ros2.repos ./
   COPY excluded-pkgs.txt ./
 
+  # this ensure the vcs import and export results are not cached
+  RUN --no-cache echo "Cloning spaceros repo artifacts"
```

Comment:
question: will this result in non-caching of everyting happing in the build after this point? i.e. will 95%+ of the build steps have to be re-run?

Reply:
Because we're only copying artifacts out of this layer, the cache *shouldn't change for downstream targets unless the actual artifacts have changed.